### PR TITLE
Add InSpec 1.0.0

### PR DESCRIPTION
**Pull request by jvrplmlmn**
_If there’s a checkbox you can’t complete for any reason, that's okay, just explain in detail why you weren’t able to do so._

After making all changes to the cask:
- [x] `brew cask audit --download {{cask_file}}` is error-free.
- [x] `brew cask style --fix {{cask_file}}` reports no offenses.
- [x] The commit message includes the cask’s name and version.

Additionally, **if adding a new cask**:
- [x] Named the cask according to the [token reference](https://github.com/caskroom/homebrew-cask/blob/master/doc/cask_language_reference/token_reference.md).
- [x] `brew cask install {{cask_file}}` worked successfully.
- [x] `brew cask uninstall {{cask_file}}` worked successfully.
- [x] Checked there are no [open pull requests](https://github.com/caskroom/homebrew-cask/pulls) for the same cask.
- [x] Checked that the cask was not already refused in [closed issues](https://github.com/caskroom/homebrew-cask/issues?q=is%3Aissue+is%3Aclosed).

### CODE REVIEW

File: Casks/inspec.rb [hunk 1]
```diff
@@ -0,0 +1,25 @@
+cask 'inspec' do
+  version '1.0.0-1'
+  sha256 '6bf1674a461cbc616b5e91ea5b426c226b00ffda7082add155bd6d0d1b33ba9d'
+
+  # packages.chef.io was verified as official when first introduced to the cask
+  url "https://packages.chef.io/stable/mac_os_x/10.11/inspec-#{version}.dmg"
+  appcast 'https://github.com/chef/inspec/releases.atom',
+          checkpoint: 'c9905934b7e515e3f0486187dabee2bb43dc7a63677f0fdc0ca85bc0f316597a'
+  name 'InSpec'
```
Comment (vitorgalvao) [inline Casks/inspec.rb]: Remove this one.

File: Casks/inspec.rb [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+  # When updating this cask, please verify the list of paths to delete (and
+  # add to it if necessary):
+  #  find /usr/local/bin -lname '/opt/inspec/*'
+  # (Source: https://docs.chef.io/install_dk.html#mac-os-x)
+  uninstall pkgutil: 'com.getchef.pkg.inspec',
+            delete:  [
+                       '/opt/inspec/',
+                       '/usr/local/bin/inspec',
+                     ]
+  zap delete: '~/.inspec/'
```
Comment (vitorgalvao) [inline Casks/inspec.rb]: This did not pass `style`. It needs an extra line above.

File: Casks/inspec.rb [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+  url "https://packages.chef.io/stable/mac_os_x/10.11/inspec-#{version}.dmg"
+  appcast 'https://github.com/chef/inspec/releases.atom',
+          checkpoint: 'c9905934b7e515e3f0486187dabee2bb43dc7a63677f0fdc0ca85bc0f316597a'
+  name 'InSpec'
+  name 'InSpec by Chef'
+  homepage 'http://inspec.io/'
+
+  pkg "inspec-#{version}.pkg"
+
+  # When updating this cask, please verify the list of paths to delete (and
```
Comment (vitorgalvao) [inline Casks/inspec.rb]: Remove these comments and instead add an [`uninstall_postflight`](https://github.com/caskroom/homebrew-cask/blob/master/doc/cask_language_reference/stanzas/flight.md) ([example](https://github.com/caskroom/homebrew-cask/blob/f2fcaa324c3a1f8bd9eab9261963b3ffdad10c65/Casks/virtualbox-extension-pack.rb#L22L24).

In it, give the comment “As suggested in https://docs.chef.io/install_dk.html#mac-os-x”.

**Review by vitorgalvao — CHANGES_REQUESTED** [review]

**Comment by jvrplmlmn** [comment]
Thanks for your quick response @vitorgalvao 

I've addressed your comments and fix the style issue. Still waiting for the CI to finish.

Let me know if you want me to squash all the commits if you consider this ready to merge.

**Review by vitorgalvao — APPROVED** [review]

**Comment by vitorgalvao** [comment]
Thank you for the quick changes.

No need to squash, as github now allows us to do that on merge. In fact, *not* squashing is now preferred, since it makes it easier to see the changes from the last time we looked at it.

**Pull request merged**